src/Microsoft.Crank.RegressionBot/Program.cs

```diff
@@ -161,10 +161,13 @@ private static async Task<int> Controller(BotOptions options)
                 {
                     var configuration = await LoadConfigurationAsync(configurationFilenameOrUrl);
                     sources.AddRange(configuration.Sources);
-                    
-                    foreach (var template in configuration.Templates)
+
+                    if (configuration.Templates != null)
                     {
-                        templates[template.Key] = template.Value;
+                        foreach (var template in configuration.Templates)
```

Comment:
Why doesn't this one get the same `?? Enumerable.Empty` treatment?

Reply:
It's a `Dictionary<string,string>`, `IEnumerable<KeyValuePair<string,string>>` might be too ugly